### DISCUSSION TITLE
TIL can make a i3 parent container fullscreen - game changer!

### DISCUSSION THREAD

**Discussion by sswam** [i3/i3#5501] [Show and tell]
Today I learned that we can make a parent container in i3 fullscreen - that's a game changer for me!

Example, open 4 windows, they're getting too skinny, right? Drag one down to put it under a parent with another window, then press mod+a to select the parent, then mod+f and mod+e. Now your two selected windows are fullscreen.

I've only just noticed that we can do this, but I'm sure it's going to be very useful!

I suppose this trick is probably mentioned in the manual, which I confess I have not yet read in full detail.

Hope this is useful to others.

**Comment by dwedia** [comment]
Nice trick, thanks

**Comment by lkuty** [comment]
> Drag one down to put it under a parent with another window,

You mean that it is a floating window ?
